src/Components/CountdownTimer/CountdownTimer.js

```diff
@@ -0,0 +1,136 @@
+import React, { useContext } from "react";
+import { SettingsContext } from "../../Context/SettingsContext";
+import { ViewContext } from "../../Context/ViewContext";
+import Timer from "react-compound-timer";
+import style from "./CountdownTimer.module.scss";
+import playTimerIcon from "../../assets/timer/playTimerIcon.png";
+import pauseTimerIcon from "../../assets/timer/pauseTimerIcon.png";
+import skipTimerIcon from "../../assets/timer/skipTimerIcon.png";
+import resetTimerIcon from "../../assets/timer/resetTimerIcon.png";
+
+const CountdownTimer = () => {
+  const [ settings ] = useContext(SettingsContext)
+  const [ view, setView ] = useContext(ViewContext)
+
+  const timerDone = () => {
+    setView('content-selection')
+  }
+
+  const countdownColorChange = (color) => {
+    const circlePath = document.getElementById('base-timer-path-remaining')
+    circlePath.style.stroke = color
+  }
+
+  return (
+    <div className={style.countdownTimer}>
+      <h2 className={style.prompt}>Click ▶ to Begin Focusing</h2>
+      <Timer
+        direction="backward"
+        initialTime={settings.workInterval * 60000}
+        lastUnit="m"
+        startImmediately={false}
+        timeToUpdate={10}
+        checkpoints={[
+          {
+            time: 0,
+            callback: () => timerDone(),
+          },
+          /* These colors can be found in our variables.scss file (salmon progression) */
+          {
+            time: settings.workInterval * 45000,
+            callback: () => countdownColorChange("#e19e9b"),
+          },
+          {
+            time: settings.workInterval * 30000,
+            callback: () => countdownColorChange("#f06b64"),
+          },
+          {
+            time: settings.workInterval * 15000,
+            callback: () => countdownColorChange("#e44f47"),
+          },
+        ]}
+      >
+        {({ start, pause, reset, getTime}) => (
```

Comment:
Such a bummer that we couldn't get the response to work with our visual countdown functionality... I'm confident we can figure out a way to get this working though!